src/System.CommandLine/Option.cs

```diff
@@ -105,12 +126,21 @@ internal virtual Argument Argument
         }
 
         //TODO: Guard against Argument.None?
```

Comment:
Out of scope of this PR, but this comment appear to be related to the Arity property and not this one.

Reply:
Is this worth opening an issue? Is there a test we can write that will demonstrate the problem this comment is pointing out?